src/sentry/static/sentry/app/components/asyncComponent.tsx

```diff
@@ -115,7 +115,7 @@ export default class AsyncComponent<
     }
   }
 
-  UNSAFE_componentWillMount() {
+  componentDidMount() {
     this.api = new Client();
```

Comment:
Not a blocker, but it's much safer to initiate `api` in the constructor as client is passed down and consumed by its children.